src/main/java/com/pirimid/cryptotrade/CryptoTradeApplication.java

```diff
@@ -9,23 +9,26 @@
 
 import java.sql.Date;
 import java.time.Instant;
+import java.util.Optional;
 
 @SpringBootApplication
 public class CryptoTradeApplication {
     @Autowired
     private UserRepository userRepository;
-    private User user=null;
     @Bean
     public User getUser(){
```

Comment:
can shift this logic to seprate userService instead of writting in main class?

Reply:
this is just to create one demo user to work with, we will remove it later on.

Reply:
we can move this code to bootstrap file